src/applications/personalization/dashboard/components/benefit-payments-and-debts/BenefitPaymentsAndDebt.jsx

```diff
@@ -19,14 +20,30 @@ const NoRecentPaymentText = () => {
   );
 };
 
-const BenefitPaymentsAndDebt = ({ payments, debts, debtsError }) => {
+const BenefitPaymentsAndDebt = ({
+  payments,
+  debts,
+  debtsError,
+  shouldShowLoadingIndicator,
+}) => {
   const lastPayment =
     payments
       ?.filter(p => moment(p.payCheckDt) > moment().subtract(31, 'days'))
       .sort((a, b) => moment(b.payCheckDt) - moment(a.payCheckDt))[0] ?? null;
 
   const debtsCount = debts?.length || 0;
 
+  if (shouldShowLoadingIndicator) {
+    return (
+      <div className="vads-u-margin-y--6">
+        <h2 className="vads-u-margin-top--0 vads-u-margin-bottom--2">
+          Benefit payments and debts
+        </h2>
+        <va-loading-indicator message="Loading benefit payments and debt..." />

```

Comment:
Should this be 'debts' instead of 'debt"? Not sure of the preferred language but just noticed it says debts above in the heading 🤷🏼

Reply:
fair enough. Will fix that now

Reply:
fixed